jenkins-integration/beaker/gatling/10_start_gatling_scenario.rb

```diff
@@ -0,0 +1,35 @@
+test_name "Start gatling scenario"
+
+simulation_id = ENV['GATLING_SIMULATION_ID']
+gatling_scenario = ENV['GATLING_SCENARIO']
+sut_hostname = ENV['GATLING_SUT_HOSTNAME']
+
+if not simulation_id
+  abort "GATLING_SIMULATION_ID environment variable required"
+elsif not gatling_scenario
+  abort "GATLING_SCENARIO environment variable required"
+elsif not sut_hostname
+  abort "GATLING_SUT_HOSTNAME environment variable required"
+end
+
+repo_url = "https://github.com/puppetlabs/gatling-puppet-load-test.git"
+repo_path = "/root/gatling-puppet-load-test_#{simulation_id}"
```

Comment:
Yeah the results get stored in the `simulation-runner/results` directory, and it seems easier to separate them out this way. It could be the case (probably is) that it wouldn't cause issues if they all ran from the same directory, but this is how we've been doing it manually so far